index.js

```diff
@@ -176,18 +176,18 @@ function getExistingPath (command, opts) {
 
 module.exports._getNpmCache = getNpmCache
 function getNpmCache (opts) {
-  return which(opts.npm).then(npmPath => {
-    const args = ['config', 'get', 'cache', '--parseable']
+  return which('node').then(nodePath => {
+    const args = [opts.npm, 'config', 'get', 'cache', '--parseable']
```

Comment:
Oh, makes sense. What about we run it through this way only for windows and only if `opts.npx` is absolute and/or ends with `.js`?